src/app/components/mnoe-tasks/mnoe-tasks.coffee

```diff
@@ -209,7 +209,7 @@
 
     # Formats dates yesterday & beyond differently from today
     expandingDateFormat = (value)->
-      dateFormat = if moment(value) < moment().startOf('day') then 'MMMM d' else 'h:mma'
+      dateFormat = if moment.utc(value) < moment().startOf('day') then 'MMMM d' else 'h:mma'
```

Comment:
I think you could use moment's `calendar()` function. The comment is not really clear about what is done here. Couldn't it be simplified?